modules/Emails/Email.php

```diff
@@ -4372,11 +4372,9 @@ private function sendOptInEmailToEmailAddressById($id, $sendOptInCheckbox = null
             /** @var \EmailAddress $emailAddress */
             $emailAddresses = BeanFactory::getBean('EmailAddresses');
             $emailAddress = $emailAddresses->retrieve($id);
-            if (
-                ($emailAddress->confirm_opt_in != '1' && empty($emailAddress->opt_in_email_created))
-                || $sendOptInCheckbox
```

Comment:
`$sendOptInCheckbox` is unused. If you removed the `send_opt_in_checkbox` from from-end you also should remove the test of it from back-end because it is unused.

Reply:
I have updated this to not used have the send_opt_in_checkbox